src/loaders/genome_collection/compute_genome_taxa_count.py

```diff
@@ -34,6 +34,9 @@
   --root_dir ROOT_DIR   Root directory for the collections project. (default: /global/cfs/cdirs/kbase/collections)
   --input_source {GTDB,genome_attributes}
                         Input file source
+  --file_group FILE_GROUP
+                        Assign file group (default: File group permission for created
+                        data files. Default is kbase)
```

Comment:
There should probably be a way to say "don't change the file group"

Reply:
👍

Reply:
Is `None` a legal group name?

Reply:
yea. it's technically legal.

Reply:
Hm, is there a way to make the argument an illegal name?

Reply:
I don't have permissions to try to create users group and test illegal user group names in NERSC. I think it's pretty safe to use None here. In order for this to break, someone (Shane) will actually create a users group called None and make it the default users group for our project.

Reply:
$ sudo groupadd no:group
groupadd: 'no:group' is not a valid group name

Reply:
👍 Set KEEP_FILE_GROUP = 'no:group'. Did you test it on NERSC? That user group is completely legal in mac.

Reply:
No, just my laptop. Let's see if @scanon has admin on NERSC